docs/user-quick-start.md

```diff
@@ -0,0 +1,299 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quick start guide for testing application DR
+
+This page will help you to set up an environment for testing **Regional-DR** basic tests.
+
+## What you’ll need
+
+To set up a *Ramen* development environment you will need to run 3
+Kubernetes clusters. *Ramen* makes this easy using minikube, but you need
+enough resources:
+
+- Bare metal or virtual machine with nested virtualization enabled
+- 8 CPUs or more
+- 20 GiB of free memory
+- 100 GiB of free space
+- Internet connection
+- Linux - tested on *Fedora* 37 and 38
+- non-root user with sudo privileges (all instructions are for non-root user)
+
+## Setting up your machine
+
+1. Fork the *Ramen* repository at https://github.com/RamenDR/ramen
+
+1. Clone the source locally
+
+   ```
+   git clone https://github.com/<your_github_id>/ramen.git
+   cd ramen
+   ```
+
+1. Create python virtual environment
+
+   The *Ramen* project use python tool to create and provision test
+   environment and run tests. The create a virtual environment including
+   the tools run:
+
+   ```
+   make venv
+   ```
+
+   This create a virtual environment in `~/.venv/ramen` and a symbolic
+   link `venv` for activating the environment. To activate the
+   environment use:
+   
+   ```
+   source venv
+   ```
+   
+   To exit virtual environment type *deactivate*.
+
+1. Add yourself to the libvirt group (required for minikube kvm2 driver).
+
+   ```
+   sudo usermod -a -G libvirt $(whoami)
+   ```
+
+   Logout and login again for the change above to be in effect.
+
+1. Install minikube - on Fedora you can use::
+
+   ```
+   sudo dnf install https://storage.googleapis.com/minikube/releases/latest/minikube-latest.x86_64.rpm
+   ```
+   
+   Tested with version v1.31.
+   
+1. Install `@virtualization` group - on Fedora you can use:
+
+   ```
+   sudo dnf install @virtualization
+   ```      
```

Comment:
It will be better to install the virtualization group before, since it is required by minikube. Practically it works in any order, but it makes sense to list the requirements this way.

Will be good to add link to Fedora docs about virtualization:
https://docs.fedoraproject.org/en-US/quick-docs/virtualization-getting-started/